### ISSUE TITLE
Unable to save form when replacing field with same handle

### ISSUE THREAD

**Issue by chrismlusk** [verbb/formie#1854]
### Describe the bug

I have an existing form with a field that I need to change its field type (Number → Single-line Text). Since I can't change the type of the current instance, I delete it and add a new field to the layout and reconfigure the field settings; the handle does not change. When I try to save the form, I get an error because that handle has already been taken.

### Steps to reproduce

1. Edit an existing form
2. Delete a field
3. Add a new field to the field layout and give it the same handle
4. Save

### Craft CMS version

4.8.10

### Plugin version

2.1.12

**Comment by engram-design** [comment]
Yep, a bit of a trick to this one that it needs to be a two-step process - delete the field, save, then add the new field.

**Comment by engram-design** [comment]
Updated in [4.0.0-beta.3](https://github.com/verbb/formie/releases/tag/4.0.0-beta.3).
